vscode-extension/src/extension.ts

```diff
@@ -38,8 +38,8 @@ export function activate(context: vscode.ExtensionContext) {
     // The commandId parameter must match the command field in package.json
     disposables.push(vscode.commands.registerCommand('ilspy.decompileAssemblyInWorkspace', async () => {
         // The code you place here will be executed every time your command is executed
-        const assembly = await pickAssembly();
-        await decompileFile(assembly);
+        const assembly = await pickAssembly() as AssemblyQuickPickItem;
```

Comment:
You can return `AssemblyQuickPickItem` from `pickAssembly()` then casting is not needed.
```suggestion
        const assembly = await pickAssembly();
```